src/sentry/api/endpoints/project_details.py

```diff
@@ -223,6 +227,10 @@ def put(self, request, project):
             project.name = result['name']
             changed = True
 
+        if result.get('platform'):
```

Comment:
Proposed solution for this is to add a function that pulls in the json to constants that can be used by the two relevant endpoints. It works locally but is failing on Travis. I'm working with Max on debugging this but if it takes more than another 30mins in the morning i'll ask for a little consultation time @mattrobenolt. (Unless there is another travis-friendly way to go about pulling in this json).